cross/nettle/Makefile

```diff
@@ -33,3 +33,6 @@ ifeq ($(findstring $(ARCH), $(x64_ARCHES)),$(ARCH))
 CONFIGURE_ARGS = --target=x86_64-pc-linux-gnu --host=x86_64-pc-linux-gnu
 endif
 CONFIGURE_ARGS += --prefix=$(INSTALL_PREFIX) --enable-shared
+
+# Fix for evansport builds to find gmp libraries
```

Comment:
I really have to understand why on hearth evansport is the only one to fail !

Reply:
I'm pretty sure this has nothing to do with us / our framework. See here:
http://lists.lysator.liu.se/pipermail/nettle-bugs/2017/006951.html
This was also the source of inspiration for the fix.

Reply:
These libraries gmp, mpfr, mpc are lacking proper support for cross-compiling... does not generate pkgconfig files, and I wonder when it expects to honor prefix when you read `spk/ffmpeg/work-evansport-5.2/install/var/packages/ffmpeg/target/lib/libgmp.la`
```
# Directory that this library needs to be installed in:
libdir='/usr/local/lib'
```
Really ?

Reply:
:-)